izer/izer.py

```diff
@@ -521,7 +523,7 @@ def main():
             eprint(f'{layer_pfx(ll)}Pooling or zero-padding results in a zero data '
                    f'dimension (input {input_dim[ll]}, result {pooled_dim[ll]}).')
 
-        if operator[ll] != op.CONV1D:
+        if not (operator[ll] == op.CONV1D or (operator[ll] == op.NONE and input_dim[ll][1] == 1)):
```

Comment:
if operator[ll] != op.CONV1D and (operator[ll] != op.NONE or input_dim[ll][1] != 1):

Reply:
changed